.github/workflows/release.yml

```diff
@@ -5,17 +5,29 @@ on:
     branches: main
 
 jobs:
-  release:
+  tag:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-
+      - uses: actions/checkout@v4
       - id: tag
         uses: freckle/haskell-tag-action@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      tag: ${{ steps.tag.outputs.tag }}
+
+  release:
+    needs: tag
+    if: needs.tag.outputs.tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: sudo apt-get install --assume-yes --no-install-recommends librdkafka-dev
+      - uses: freckle/stack-action@v5
+        with:
+          test: false
+          stack-build-arguments: --haddock --haddock-for-hackage
 
-      - if: steps.tag.outputs.tag
-        uses: freckle/stack-upload-action@v2
+      - run: |
+          stack --stack-yaml stack-lts-18.28.yaml upload --pvp-bounds lower .
```

Comment:
Why such an old resolver here?

Reply:
I actually meant to comment this and will now, but:

The `--pvp-bounds lower` option uses the resolver to decide the lower bounds. Since this is the oldest resolver we test with, the versions of the packages in this resolver are exactly what we want to be our lower bounds.

https://illuminate.atlassian.net/wiki/spaces/EN/pages/17185976662/Preferred+Supported+Compilers+in+Haskell+OSS#Lower-Bounds